ansible/join-cluster.yml

```diff
@@ -95,3 +95,12 @@
       command: "kubeadm join --config /etc/kubeadm/kubeadm-join.yaml"
       args:
         creates: /etc/kubernetes/kubelet.conf
+
+    - name: Render azure config for api-server
+      template:
+        src: azure.json.j2
+        dest: /etc/kubernetes/kubelet-azure.json
+        owner: root
+        group: root
+        mode: '0644'
+      when: cloud_provider is defined and cloud_provider == "azure"
```

Comment:
This seems to be a duplicate from `prepare-nodes.yml`.
```suggestion
```

Reply:
But that is only masters. This is for the workers

Reply:
Ah I see! Would it then make more sense to move this to the `k8s_node` role to avoid duplication?

Reply:
Yea, that would probably make more sense.